src/function/built_in_functions.cpp

```diff
@@ -60,24 +60,24 @@ void BuiltInFunctions::registerAggregateFunctions() {
     registerCollect();
 }
 
-ScalarFunction* BuiltInFunctions::matchScalarFunction(
-    const std::string& name, const std::vector<LogicalType>& inputTypes) {
+Function* BuiltInFunctions::matchScalarFunction(
```

Comment:
should we unify matchAggregateFunction as well?

Reply:
I have a todo here. Aggregate function involves checking for distinct keyword, so it is a bit complicated.